delivery/exceptions.py

```diff
@@ -87,3 +87,24 @@ class CannotParseDDSOutputException(Exception):
     project.
     """
     pass
+
+
+class RequiredFileNotFoundException(Exception):
```

Comment:
You're right, it's probably a bit over ambitious to create custom exceptions for trivial cases.

Reply:
Actually maybe it would make sense to have one exception type per http error code that we use (403, 404 and 500). Then you could either create three custom ones or use the built-in exceptions `FileNotFoundError`, `PermissionError` and `RuntimeError`.